app/models/organization.rb

```diff
@@ -264,6 +267,14 @@ def earliest_reporting_year
     year
   end
 
+  def opt_in_email_notification
+    update(email_notification_opt_in: true)
+  end
+
+  def opt_out_email_notification
+    update(email_notification_opt_in: false)
+  end
```

Comment:
You don't need special methods to change this attribute, you can do simple `organization.email_notification_opt_in = true` etc.